### ISSUE TITLE
[junying/event_filter]: logs["success"] always be true even though  evm reverted the transaction

### ISSUE THREAD

**Issue by youngqqcn** [orientwalt/htdf#49]
code: bc17054f9c71124fc6a59cd1c0c84dc0460cc21d
test case:   pythontests/test_htdf_faucet_contract.py

```json
$ hscli query tx 2DD812A5449C4D43A9506F5DBD8C9D38CF4273A515A57DFB80BFE955D6E751CC
{
  "height": "160",
  "txhash": "2DD812A5449C4D43A9506F5DBD8C9D38CF4273A515A57DFB80BFE955D6E751CC",
  "data": "BB020A140000000000000000000000000000000000000000128002000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002A202DD812A5449C4D43A9506F5DBD8C9D38CF4273A515A57DFB80BFE955D6E751CC",
  "raw_log": "[{\"msg_index\":\"0\",\"success\":true,\"log\":\"executed EVM state transition; sender address: 0x39d626af96eF898D1a76dA9e161eA98D90BB89f4, recipient address: 0x578a596c71fB26F68f490302B851B752342D62D8, output: , err: evm: execution reverted\"}]",
  "logs": [
    {
      "msg_index": "0",
      "success": true,  /////////////////////////////////////  ============== HERE
      "log": "executed EVM state transition; sender address: 0x39d626af96eF898D1a76dA9e161eA98D90BB89f4, recipient address: 0x578a596c71fB26F68f490302B851B752342D62D8, output: , err: evm: execution reverted" //////////////////// ====== HERE
    }
  ],
....

```

**Comment by youngqqcn** [comment]
the `err` of `evm.Call` returns is not be handled 

https://github.com/orientwalt/htdf/blob/bc17054f9c71124fc6a59cd1c0c84dc0460cc21d/x/evm/types/transition.go#L306-L333

**Comment by bluesky0125** [comment]
this issue resolved since commit: 1bfc187815b0fabfcfb27cfccfa4eba1b833d760

**Comment by bluesky0125** [comment]
this issue resolved since commit: 1bfc187815b0fabfcfb27cfccfa4eba1b833d760
